docs/advanced_topics/third_party_tutorials.rst

```diff
@@ -19,6 +19,7 @@ Third-party tutorials
 * `Wagtail and GraphQL <https://jossingram.wordpress.com/2018/04/19/wagtail-and-graphql/>`_ (19 April 2018)
 * `Wagtail and Azure storage blob containers <https://jossingram.wordpress.com/2017/11/29/wagtail-and-azure-storage-blob-containers/>`_ (29 November 2017)
 * `Building TwilioQuest with Twilio Sync, Django [incl. Wagtail], and Vue.js <https://www.twilio.com/blog/2017/11/building-twilioquest-with-twilio-sync-django-and-vue-js.html>`_ (6 November 2017)
+* `Image upload in wagtail forms<https://posts-by.lb.ee/image-uploads-in-wagtail-forms-3121c9b35d27>`_  Tutorial on adding image uploads to wagtail forms(3 September 2017) 
```

Comment:
@a-mere-peasant  - thanks for getting this added, below is my comment from the other PR - minor fixes needed please.

Just a few typos here, there should be a space just before the opening bracket of the link, also there should be a space before the opening bracket of the date (to align with the others that have been added.

If you can fix those minor things up I will get this merged in asap.

Reply:
Okay, I'll fix them up, thanks for letting me help.

Reply:
@lb- should i also remove the description after the link since others don't have one or is it okay to keep it ?

Reply:
Hey, yeah probably, just to match the existing convention. thanks again :D

Reply:
I've changed the file accordingly, please tell me if something else has to be done, Happy to help!